### PR TITLE
Matthew's pull request

### PR DESCRIPTION
**Pull request by matttunwashe**
This is the pull request for my responsive webpage.

### CODE REVIEW

File: about-me.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+	   <title> Responsive Navigation </title> <!-- what I called my website title -->
```
Comment (JSChilds) [inline about-me.html]: This is too far indented, delete some of this space.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Improve on the indentation and spacing issues to make the code easier to read

File: about-me.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+	   <title> Matthew Tunwashe Portfolio </title> <!-- website title -->
```
Comment (JSChilds) [inline about-me.html:5]: Try and keep your spacing consistent from now on so the code flows cleanly and is easy to read.

**Review by JSChilds — CHANGES_REQUESTED** [review]
Great effort, just work on fixing your spacing and indentation before you add anymore to it.

File: about-me.html (hunk 2 above)
Comment (matttunwashe) [reply about-me.html:5]: Feedback noted and corrections has been made.

**Review by JSChilds — APPROVED** [review]
Some things still a little too indented, but definitely looking better, good effort. Try to keep an eye on your indentation as you're going to make sure it doesn't get out of line and maybe look at somebody else's HTML to get a sense of how it should look.

**Pull request merged**